package/gluon-core/files/lib/gluon/upgrade/040-site-code

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/lua
+
+local sysconfig = require 'gluon.sysconfig'
+local site = require 'gluon.site_config'
+
+sysconfig.site_code = site.site_code
```

Comment:
This piece of code was added before we decided to convert site.conf to JSON (so creating a sysconfig file was the only way to get the site code from C). I forgot to remove it when rebasing...